46.md

```diff
@@ -0,0 +1,159 @@
+NIP-46
+======
+
+Nostr Connect
+------------------------
+
+`draft` `optional` `author:tiero` `author:giowe` `author:vforvalerio87`
+
+## Rationale
+
+Private keys should be exposed to as few systems - apps, operating systems, devices - as possible as each system adds to the attack surface.
+
+Entering private keys can also be annoying and requires exposing them to even more systems such as the operating system's clipboard that might be monitored by malicious apps.
+
+
+## Terms
+
+* **App**: Nostr app on any platform that *requires* to act on behalf of a nostr account.
+* **Signer**: Nostr app that holds the private key of a nostr account and *can sign* on its behalf.
+
+
+## `TL;DR`
+
+
+**App** and **Signer** sends ephemeral encrypted messages to each other using kind `24133`, using a relay of choice. 
+
+App prompts the Signer to do things such as fetching the public key or signing events.
+
+The `content` field must be an encrypted JSONRPC-ish **request** or **response**.
```

Comment:
Why not just define that this is json-rpc?

Reply:
Already discusses. we just using request and response, all the rest does not conform to JSONRPC which assume TCP channels/HTTP as transport

Reply:
Ah ok, sorry I didn't saw that discussion - I will try to find that... I think JSONRPC is transport agnostic and would be a perfect match here - it also just defines a request- & response object.. see https://www.jsonrpc.org/specification